src/components/logoCard/tests/LogoCard.spec.tsx

```diff
@@ -0,0 +1,153 @@
+import { mount, ReactWrapper, shallow } from 'enzyme';
+import * as React from 'react';
+import * as _ from 'underscore';
+import { CornerRibbon, DEFAULT_CORNER_RIBBON_CONTAINER_CLASSNAME, PlacementX, PlacementY } from '../../cornerRibbon/CornerRibbon';
+import { DEFAULT_LOGO_CARD_CLASSNAME, DEFAULT_LOGO_ICON, DEFAULT_LOGO_ICON_CLASSNAME, DEFAULT_LOGO_ICON_SIZE, ILogoCardProps, LogoCard } from '../LogoCard';
```

Comment:
I think this line might be a little long. Does `Optimize imports` re-format your imports on multiple lines?

Reply:
Can't seem to find an `Optimize imports` command in vscode, is this an extension?

Reply:
It is something in Webstorm, but I've managed to do the same thing in VSCode with the same keyboard shortcut. `ctrl + alt + o`. This is on Mac though and when using the typescript hero extension. The keyboard shortcut is mapped to: "typescriptHero.imports.organize".